includes/wikia/api/RevisionApiController.class.php

```diff
@@ -31,6 +33,7 @@ public function getRevisionsDiff() {
 
 		$avatar = $request->getBool( 'avatar' );
 		$oldRev = $request->getBool( 'oldRev' );
+		$upvote = $request->getBool( 'upvote' );
```

Comment:
see https://github.com/Wikia/app/pull/9784/files#diff-784f2baa3bf95847c76ff321f96a6544R101 and https://github.com/Wikia/app/commit/6530d005cec8613479ba735400a29d4a82acb738. I added needed data about revision there, so we don't need to query database (we will get data about users).